content/pages/changelog/index.md

```diff
@@ -13,6 +13,267 @@ For historical changelog entries, please see our
+events will return additional fields.
+
+### New task and task assignment events
+
+#### Task update
+
+```js
+      {
+            "source": {
+                "item_type": "file",
+                "item_id": "123",
+                "item_name": "github_PNG20 (3) (2).png",
+                "parent": {
+                    "type": "folder",
+                    "name": "2020-02-27",
+                    "id": "1234"
+                },
+                "owned_by": {
+                    "type": "user",
+                    "id": "12345",
+                    "name": "Jane Doe",
+                    "login": "email@example.com"
+                }
+            },
+            "created_by": {
+                "type": "user",
+                "id": "12345",
+                "name": "Jane Doe",
+                "login": "email@example.com"
+            },
+            "action_by": null,
+            "created_at": "2020-04-02T22:26:55-07:00",
+            "event_id": "66585c35-97a8-4882-9fec-ce7e178b2e53",
+            "event_type": "TASK_UPDATE",
+            "type": "event",
+            "additional_details": {
+                "size": 67801,
+                "task": {
+                         "isCompleted": false,
+                         "due": "2020-07-06T10:49:44-07:00"
+"id": "1234567"
+"description": "task description"
+"creatorId": "567890"
+                },
+                "version_id": "26954851916",
+                "service_id": "12345",
+                "service_name": "Another App"
+            }
+        }
+```
+
+#### Task assignment delete
+
+```js
+        {
+            "source": {
+                "item_type": "file",
+                "item_id": "123",
+                "item_name": "football.png",
+                "parent": {
+                    "type": "folder",
+                    "name": "All Files",
+                    "id": "0"
+                },
+                "owned_by": {
+                    "type": "user",
+                    "id": "12345",
+                    "name": "Jane Doe",
+                    "login": "email@example.com"
+                }
+            },
+            "created_by": {
+                "type": "user",
+                "id": "12345",
+                "name": "Jane Doe",
+                "login": "email@example.com"
+            },
+            "action_by": null,
+            "created_at": "2020-05-28T15:53:28-07:00",
+            "event_id": "88c0f6fd-e644-468e-aac3-d8ad7fba4eff",
+            "event_type": "TASK_ASSIGNMENT_DELETE",
+            "ip_address": "Unknown IP",
+            "type": "event",
+            "session_id": null,
+            "additional_details": {
+                "size": 5875,
+                "version_id": "712094977889",
+                "service_id": "123",
+                "service_name": "Box Elements"
+            }
+        },
+```
+
+### Updated task and task assignment events with new fields
```

Comment:
Do you think it might be useful for people to know what changed here? Do we have this information?

Reply:
Yes we do, it is in the collab ticket I shared with you. How do you want that represented?

Reply:
We could extract the full details of the Task events into its own page, like we did with the Shield event. 

We can then make this changelog something like:


---

Starting today, the [enterprise event stream](.) will
begin producing new task and task assignment events, and some existing task
events will return additional fields.

### New events

* When a task is updated this will now create a `TASK_UPDATE` event
* When a task is delete this will now create a `TASK_DELETE` event
* ....

### Updated events

* When a task is created, the `TASK_CREATE` event will now include the additional fields `foo` and `bar` in the list of `additional_fields` 
* ....

See the [task events](.) documentation for more information on the payloads produced within these events.
---